lib/ws_servers/api/algos/algo_worker.js

```diff
@@ -24,25 +28,39 @@ class AlgoWorker {
     this.marketData = marketData
   }
 
-  async start (apiKey, apiSecret, userID) {
-    const { dms, affiliateCode, wsURL, restURL } = this.settings
+  async start ({ apiKey, apiSecret, authToken, userId }) {
+    const { dms, affiliateCode, wsURL, restURL, plugins = [] } = this.settings
 
-    this.userID = userID
+    this.userId = userId
 
     d(
       'spawning bfx algo host (dms %s) [aff %s]',
       dms ? 'enabled' : 'disabled',
       affiliateCode
     )
 
+    if (!authToken) {
```

Comment:
in case if authToken is passed do we handle token renewal somewhere else?

Reply:
Yes, is handled by `TokenRenewalPlugin` below https://github.com/bitfinexcom/bfx-hf-server/pull/102/files#diff-b69b69ff8ac30d9313bf79cf719aaf18f222d9aebc9d2ddc1e03ec20b9a64fa1R52